### ISSUE TITLE
View with definition including 

### ISSUE THREAD

**Issue by JohnL4** [littleK0i/SnowDDL#147]
**Describe the bug**
A description of what the bug is.

I suspect the problem is in the greediness of this regexp: https://github.com/littleK0i/SnowDDL/blame/c7ca3c48a533340bc88ac690b6664840005093ec/snowddl/converter/view.py#L11

**Expected behavior**
A description of what you expected to happen.

**Attach log**
Attach log produced by SnowDDL CLI interface or by your application. Please make sure to include "context" information which is normally produced at the beginning of the log.

**Attach YAML config (if applicable)**
If you have a config with minimal reproducible example, please attach it as well.

**Comment by littleK0i** [comment]
Wish we could find a way to extract VIEW text without relying on parsing. Maybe new REST API endpoints can do that, I'll take a look at some point.

Otherwise the only way to do it reliably is to introduce actual SQL parsing.

**Comment by JohnL4** [comment]
Yeah, I hear you. Was thinking of that StackOverflow post on parsing HTML with regexps. (And, yes, parsing SQL is Hard.)

**Comment by JohnL4** [comment]
Ok, so, for what it's worth, making the following change to regexp solved my one particular issue for this one particular view:

```
view_text_re = compile(r"^.*?\sas(\n|\s)+(.*)$", DOTALL)
```

(Changed ``*`` operator to ``*?``.)

No telling what other damage I've caused. :)

**Comment by littleK0i** [comment]
Upd in 0.36.0: applied your change, regexp looking for ` as` keyword is now non-greedy. 

Naturally, it may still fail if ` as` is being used somewhere in the comments for individual columns or view itself.

I'll experiment with pyparsing soon and see if we can introduce more permanent solution.

**Comment by JohnL4** [comment]
Fwiw, pyparsing docs:

ignore(expr) - function to specify parse expression to be ignored while matching defined patterns; can be called repeatedly to specify multiple expressions; useful to specify patterns of comment syntax, for example

**Comment by JohnL4** [comment]
Per note on #149 , this can be closed.
